mobile-widgets/qml/MapWidget.qml

```diff
@@ -69,17 +69,10 @@ Item {
 		ParallelAnimation {
 			id: mapAnimation
 			CoordinateAnimation {
-				target: map
-				property: "center"
-				to: map.newCenter
-				duration: 2000
+				target: map; property: "center"; to: map.newCenter; duration: 2000;
```

Comment:
it one of the few last patches i make most animations to be on single lines, so maybe i need to expand all properties to be on separate lines everywhere.